support-frontend/assets/components/subscriptionCheckouts/address/postcodeFinder.tsx

```diff
@@ -114,7 +114,10 @@ function AddressSelector({ results, onAddressSelected }: AddressSelectorProps) {
 			<Option value={''}>Select an address</Option>
 			<>
 				{results.map((result, index) => (
-					<Option key={result.lineOne} value={index}>
+					<Option
+						key={`${result.lineOne ?? ''}/${result.lineTwo ?? ''}`}
```

Comment:
We needed to be more specific here as I was getting the "Two children with the same key" error.